src/main/java/alisa/Parser.java

```diff
@@ -0,0 +1,45 @@
+package alisa;
+
+import alisa.command.*;
```

Comment:
Following the coding standard it would be better to list imports explicitly instead of using wildcards.

Reply:
Oh nice I didn't notice that, thanks!